F2Py.cmake

```diff
@@ -171,6 +171,10 @@ function (f2py_add_module target_name)
     PRIVATE ${F2PY_ADD_MODULE_COMPILE_DEFS})
   endif()  
   target_compile_options(${target_name} PRIVATE -cpp)
+  # Link dll statically in Windows
+  if (WIN32)
+    target_link_libraries(${target_name} PUBLIC "-static")
+  endif()
```

Comment:
This may become a problem, I read that one should not mix dynamic and static linking. But you constrain it to Windows and if it works for now, I am fine with it. I also don't have a better solution.

Reply:
Perhaps add a comment that this is bad in general?

Reply:
Yes, it is a problem with Linux and MacOS. The libraries don't link. It works only for Windows. I will add the comment in CMake.